runtime/doc/eval.txt

```diff
@@ -4053,6 +4053,8 @@ jobstart({cmd}[, {opts}])				{Nvim} *jobstart()*
 		  - width: Width of the terminal screen(only if pty is set)
 		  - height: Height of the terminal screen(only if pty is set)
 		  - TERM: $TERM environment variable(only if pty is set)
+		  - ignore_stdin: If set, the job will not open an stdin
+		    channel (and jobsend() won't work).
```

Comment:
Does it accept any value other than "ignore" or "pipe"? Having an "ingore_stderr" option would be the same as not associating a callback and adds another error condition to check for. To set up a pipe without actually reading it, one can just pass a `noop` callback.